app/models/debate.rb

```diff
@@ -11,23 +11,30 @@ class Debate < ApplicationRecord
 
   accepts_nested_attributes_for :answers
   validates :topic, presence: true
-  validates :closed_at, presence: true
   validates :code, presence: true, uniqueness: true,
                    length: { is: CODE_LENGTH }, numericality: { only_integer: true }
 
   before_save  :block_code_change
 
-  attribute :closed_at, :datetime, default: -> { Time.current + 1.hour }
+  attribute :is_closed, :boolean, default: false
+  alias_attribute :closed?, :is_closed
```

Comment:
If you name this attribute / column `closed` instead of `is_closed` then the rails will automatically provide `closed?` method, making this code: `alias_attribute :closed?, :is_closed` not necessary.